frontend/src/liboperator/operator/operator.go

```diff
@@ -169,6 +174,36 @@ func deviceEndpoint(c *JSONUnix, pool *DevicePool, config apiv1.InfraConfig) {
 	}
 }
 
+func openwrt(w http.ResponseWriter, r *http.Request, pool *DevicePool) {
+	vars := mux.Vars(r)
+	devId := vars["deviceId"]
+	dev := pool.GetDevice(devId)
+	if dev == nil {
+		http.Error(w, "Device not found", http.StatusNotFound)
+		return
+	}
+
+	devInfo := dev.info.(map[string]interface{})
+	openwrtDevId, ok := devInfo["openwrt_device_id"].(string)
+	if !ok {
+		http.Error(w, "Device obtaining Openwrt not found", http.StatusNotFound)
+		return
+	}
+
+	path := vars["path"]
+	openwrtAddr, ok := devInfo["openwrt_addr"].(string)
+	if !ok {
+		http.Error(w, "Openwrt address not found", http.StatusNotFound)
+		return
+	}
+
+	url, _ := url.Parse("http://" + openwrtAddr)
```

Comment:
does http works? IIRC, only https works for luci...

Reply:
Luci with http works as default, with assuming there's no proxy extensions in chrome.